ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -13,12 +13,12 @@ import {
 import {
   Modal,
   ModalOverlay,
-  ModalContent,
-  ModalHeader,
   Button,
   ButtonVariant,
   ButtonSize,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import EditGasDisplay from '../edit-gas-display';
 
 import { I18nContext } from '../../../contexts/i18n';
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 2 37 16 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/3c48f783-0b58-41a0-b894-3d7447918486">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 2 37 29 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/6addfebd-eca7-4292-a74e-0934634a8921">